docs/resources.js

```diff
@@ -12,14 +12,31 @@ function getResources() {
 }
 
 /**
- * Always use Harvey for test cases since our tests assert against specific
- * block groups and damage points.
+ * Determines and returns the current disaster.
  * @return {string} current disaster
  */
 function getDisaster() {
+  let final_disaster;
+
+  // The order of precedence for determining the current disaster is query
+  // parameter, local storage, a test specific default and finally the base
+  // default.
   const params = new URLSearchParams(window.location.search);
-  const disaster_param = params.has('disaster') ? params.get('disaster') : disaster;
-  return inProduction() ? disaster_param : disaster;
+  if (params.has('disaster')) {
+    final_disaster = params.get('disaster');
+  } else if (localStorage.getItem('disaster')) {
+    final_disaster = localStorage.getItem('disaster');
+  } else if (!inProduction()) {
+    // Start off with Harvey for test cases since our tests assert against
```

Comment:
I think we can get rid of this by having the tests just use the Harvey url? Or you could support a cookie, although I see that localStorage is nicer.

Reply:
I just set localStorage in Cypress instead.